content/kubernetes/delete_custom_resources.md

```diff
@@ -78,6 +81,26 @@ You will also need to remove [the `namespaceSelector` section from the validatin
   kubectl get reaadb -o=jsonpath='{range .items[*]}{.metadata.name}'
   ```
 
+## Delete a remote cluster (RERC)
+
+{{<note>}}The Redis Enterprise Active-Active database (REAADB) custom resource is currently in public preview. View [Preview REAADB]({{<relref "/kubernetes/active-active/preview/">}}) for more details.{{</note>}}
+
+1. Verify the RERC you want to delete isn't listed as a particpating cluster in any REAADB resources.
+
+  If an RERC is still listed as a participating cluster in any database, the deletion will fail.
```

Comment:
The numbered list isn't rendering correctly. I think adding additional spaces to increase the indentation will fix it. This is also an issue for [Delete an Active-Active database](https://docs.redis.com/staging/DOC-2239/kubernetes/delete_custom_resources/#delete-an-active-active-database-reaadb).
```suggestion
    If an RERC is still listed as a participating cluster in any database, the deletion will fail.
```